zip_archive.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import os
+import zipfile
+
+# input the dir of folder/file
+dirPath = '/home/yanuarakhid/myfolder'
```

Comment:
Please pass `dirPath` as an argument to the script instead of having it hardcoded, or at a minimum replace the hardcoded string with an empty string so that a user can set it themselves

Reply:
oke i will update soon, thanks for your suggestion